packages/figma-utils/src/commands/import-icons.ts

```diff
@@ -0,0 +1,75 @@
+import { Command } from "commander";
+import { mkdir, writeFile } from "node:fs/promises";
+import path from "node:path";
+import { writeIconMetadata } from "../icons/generate.js";
+import { optimizeSvg } from "../icons/optimize.js";
+import { parseComponentsToIcons } from "../icons/parse.js";
+import { fetchFigmaComponents, fetchFigmaSVGs } from "../index.js";
+import { isDirectory } from "../utils/fs.js";
+
+export type ImportIconsCommandOptions = {
+  fileKey: string;
+  token: string;
+  pageId: string;
+  dir?: string;
+  metaFile?: string;
+};
+
+export const importIconsCommand = new Command("import-icons")
```

Comment:
will there be a test file for this aswell? :) like import-variables.spec.ts

Reply:
Yes, I will do this in a follow up PR to not make this one any larger. Is that fine for you?